earthspy/earthspy.py

```diff
@@ -188,6 +193,14 @@
 
         return None
 
+    def get_raster_compression(self) -> None:
+        
+        try:
+            assert self.compress_mode in ['DEFLATE','LZW','PACKBITS','JPEG',
+                                            'WEBP','LZMA','ZSTD']
+        except:
```

Comment:
you can see that a vulnerability has been automatically spotted. I think it would be better to have the following structure:

```python

if compress_mode in ['DEFLATE','LZW','PACKBITS','JPEG', 'WEBP','LZMA','ZSTD']:
   self.compress_mode = compress_mode
else:
  raise KeyError("Compression mode not found")
```

or something similar